synctl

```diff
@@ -109,38 +109,36 @@ def start(pidfile: str, app: str, config_files: Iterable[str], daemonize: bool)
         return False
 
 
-def stop(pidfile: str, app: str) -> bool:
+def stop(pidfile: str, app: str) -> int:
     """Attempts to kill a synapse worker from the pidfile.
     Args:
         pidfile: path to file containing worker's pid
         app: name of the worker's appservice
 
     Returns:
-        True if the process stopped successfully
-        False if process was already stopped or an error occured
+        process id, or 0 if the process was not running
```

Comment:
On *Linux*, pid 0 is reserved for the scheduler ... is that guaranteed more generally on other systems, though?
This is likely to be inconsequential but I would probably go for an `Option[int]` if it were me, if only because I feel it expresses the intent more clearly.

Reply:
fair enough.